src/commands/cmd_hash.cc

```diff
@@ -429,6 +430,228 @@ class CommandHRandField : public Commander {
   bool no_parameters_ = true;
 };
 
+class CommandFieldExpireBase : public Commander {
+ protected:
+  Status commonParse(const std::vector<std::string> &args, int start_idx) {
+    int idx = start_idx;
+    CommandParser parser(args, idx);
+    std::string_view expire_flag, num_flag;
+    uint64_t fields_num = 0;
+    while (parser.Good()) {
+      if (parser.EatEqICaseFlag("FIELDS", num_flag)) {
+        fields_num = GET_OR_RET(parser.template TakeInt<uint64_t>());
+        idx += 2;
```

Comment:
We don't need to record the index here, it's good to use `parse.Good()` to check if there are remaining arguments to be consumed.

Reply:
Thank for you hint. I replace it with `parser.Remains()`, please review.